### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by Smart1-hub**
Create a app.rb file that will serve as console app entry-point. It should have methods that do the following:
    List all books.
    List all people.
    Create a person (teacher or student, not a plain Person).
    Create a book.
    Create a rental.
    List all rentals for a given person id.
In main.rb define the entry point, this will be a method called main that is invoked at the end of the file. This method should do the following:
    Present the user with a list of options to perform.
    Lets users choose an option.
    If needed, ask for parameters for the option.
    Have a way to quit the app.

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Smart1-hub, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters are passing
✔️ Professional readme
✔️ Correct gitflow

## Required 
- [ ] Kindly fix the screenshot in the readme which produces an error.
- [ ] Kindly fix the error which is encountered when I run ruby `main.rb`.
Screenshot 2022-08-04 at 6 07 38 PM

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Smart1-hub — COMMENTED** [review]
Update reviewer's comments

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,116 @@
+require './book'
```
Comment (danmainah) [inline app.rb:1]: - [ ] @Smart1-hub great work implementing the changes requested but as the previous reviewer pointed out, there is an uninitialized constant error for your Nameable class, kindly check on it.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Smart1-hub , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Smart1-hub — COMMENTED** [review]
Update review

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Smart1-hub, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

## Optional: Good job implementing the changes asked previously, however, I couldn't add a student or teacher as it throws an error. It is highly recommended to fix this.

https://user-images.githubusercontent.com/70269507/182887462-93229d3a-0964-4540-930f-f1323bea20a1.mov

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**